osu.Game/Overlays/Settings/Sections/Audio/LatencySettings.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Graphics;
+using osu.Game.Configuration;
+
+namespace osu.Game.Overlays.Settings.Sections.Audio
+{
+    public class LatencySettings : SettingsSubsection
+    {
+        protected override string Header => "Latency";
+
+        [BackgroundDependencyLoader]
+        private void load(AudioManager audio, OsuConfigManager config)
+        {
+            Children = new Drawable[]
+            {
+                new SettingsSlider<int>
+                {
+                    LabelText = "Device update period",
```

Comment:
`Audio buffer length` should work fine and be understandable to users. Underlying configuration can be `LinuxAudioBufferUpdatePeriod `

Reply:
> `Audio buffer length` should work fine 

As I said before, this setting doesn't control the buffer length, does it?

> Underlying configuration can be `LinuxAudioBufferUpdatePeriod`

Okay.

Reply:
as long as there is one setting the text i specified works best to describe *to a user* what it is affecting.

we will add descriptive text later when setting supports it.